applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -842,6 +840,72 @@ static struct bt_conn_cb conn_callbacks = {
 
 };
 
+/**
+ * @brief	Callback handler for play/pause.
+ *
+ * @note	This callback is called from MCS after receiving a
+ *		command from the client or the local media player.
+ *
+ * @param[in]	play  Boolean to indicate if the stream should be resumed or paused.
+ */
+static void le_audio_play_pause_cb(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
+		playing_state = true;
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_L],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable left stream");
+			}
+		}
+
+#if !CONFIG_STREAM_BIDIRECTIONAL
+		if (audio_streams[AUDIO_CH_R].ep->status.state ==
+		    BT_AUDIO_EP_STATE_QOS_CONFIGURED) {
+			ret = bt_audio_stream_enable(&audio_streams[AUDIO_CH_R],
+						     lc3_preset_nrf5340.codec.meta,
+						     lc3_preset_nrf5340.codec.meta_count);
+
+			if (ret) {
+				LOG_WRN("Failed to enable right stream");
+			}
+		}
+#endif /* !CONFIG_STREAM_BIDIRECTIONAL */
+	} else {
+		playing_state = false;
+
+		ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
+		ERR_CHK(ret);
+
+		if (audio_streams[AUDIO_CH_L].ep->status.state == BT_AUDIO_EP_STATE_STREAMING) {
+			ret = bt_audio_stream_disable(&audio_streams[AUDIO_CH_L]);
+
+			if (ret) {
+				LOG_WRN("Failed to disable left stream");
+			}
+		}
```

Comment:
Should add warning (?) prints that streams is not in BT_AUDIO_EP_STATE_STREAMING state

Reply:
See same comment two comments up